extensions/espanso/src/index.tsx

```diff
@@ -25,14 +45,26 @@ export default function Command() {
         const { packages: packageFilesDirectory, match: matchFilesDirectory } = await getEspansoConfig();
 
         const packageMatches = getMatches(packageFilesDirectory, { packagePath: true });
-
         const userMatches = getMatches(matchFilesDirectory);
 
         const combinedMatches: NormalizedEspansoMatch[] = [...userMatches, ...packageMatches];
-
         const sortedMatches = sortMatches(combinedMatches);
 
+        const categoriesSet = new Set<string>();
+        sortedMatches.forEach((match) => {
+          const sectionKey = match.label?.includes(" > ") ? match.label.split(" > ")[0] : "Miscellaneous";
```

Comment:
I think this should split with `>` instead, but add in a `.trim`, so categories can still work if users don't want the spaces.

Also, does this handle weird cases like if the `>` is in the beginning or end of the label?